seed/views/column_mapping_presets.py

```diff
@@ -109,13 +138,26 @@ def create(self, request, pk=None):
         """
```

Comment:
This might be a frontend issue instead of a create issue, but I can't edit the SEED headers when I make a copy of the default BS preset on the settings page.

_**Note: I can make edits to BS presets that are made from the import mapping page though.**_

![Screen Shot 2020-05-13 at 2 09 31 PM](https://user-images.githubusercontent.com/30608004/81860041-63dc5700-9523-11ea-887d-5a08a282b853.png)

Reply:
great catch 👍